app/views/documents/_secondary_navigation.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= render "components/secondary_navigation", {
+  aria_label: "Document",
+  items: [
+    {
+      label: t("documents.secondary_navigation.summary"),
```

Comment:
I'm not sure these should be in the translations file based on: https://github.com/alphagov/content-publisher/blob/29f45e570e7c9c0ab965c629735aaacc07969f68/docs/adr/0004-editing-microcopy.md - have a read and see what your stance is. I don't mind too much which way we go just that we're consistent with the doc and app.